src/site/apt/index.apt

```diff
@@ -0,0 +1,154 @@
+What is this?
```

Comment:
I'm going to talk through my thoughts here : 

* I have no love of markdown - using something else works fine for me. 
* I want to encourage you to make the docs better.  From that perspective whatever format you want to use is fine by me. 
* I also want others to be able to do the same.  I hadn't heard of APT before this.  Others might not know it either and would need to learn it (though it looks easy enough). 
* I also want to be able to preview the changes on GitHub easily.  This doesn't do that - we just get text.  Bummer. 

Meh, okay, let's try this.